.github/workflows/requirements.txt

```diff
@@ -0,0 +1,18 @@
+meson==0.60.1 \
+    --hash=sha256:5add789c953d984b500858b2851ee3d7add0460cf1a6f852f0a721af17384e13 \
```

Comment:
In case I borrow this idea, how do you get the hashes?

Reply:
I used https://pypi.org/project/hashin/

Reply:
I can find 79a07aabd43f5934472d71921ef3470acbf6a719ba538743c505156fb27d62f6 with this: 
```
$ pip3 download meson
Collecting meson
  Downloading meson-0.60.1-py3-none-any.whl (837 kB)
     |████████████████████████████████| 837 kB 332 kB/s 
Saved ./meson-0.60.1-py3-none-any.whl
Successfully downloaded meson
$ pip3 hash meson-0.60.1-py3-none-any.whl
meson-0.60.1-py3-none-any.whl:
--hash=sha256:79a07aabd43f5934472d71921ef3470acbf6a719ba538743c505156fb27d62f6
```
What about 5add789c953d984b500858b2851ee3d7add0460cf1a6f852f0a721af17384e13 then?

Reply:
That's https://files.pythonhosted.org/packages/ce/4d/dfc140029e13a63af66138aa81d7d5f2327aba579f1cbbee2c3e0b2fc36b/meson-0.60.1.tar.gz

Reply:
they can be found at https://pypi.org/project/meson/0.60.1/#files. That's where pip downloads them from

Reply:
Thanks!

Reply:
@mrc0mmand I wonder if it could be merged? It kind of bothers me that the dependency I'd like to pin the most isn't still pinned :-)

Reply:
Given that in the past 10 days nobody complained, I'll go ahead and *press the button* ;-)